StitchCore/StitchCore/Source/Core/StitchClient.swift

```diff
@@ -348,7 +350,7 @@ public class StitchClient: StitchClientType {
     public func logout() -> Promise<Void> {
         if !isAuthenticated {
             printLog(.info, text: "Tried logging out while there was no authenticated user found.")
-            return Promise.init { $0.fulfill(()) }
+            return Promise.value(())
```

Comment:
[q] Do we need the extra parentheses? Looks a bit odd to the eye.

Reply:
yes, you need some argument value, the value is a `Void`